consumer/consumer.go

```diff
@@ -68,3 +68,11 @@ type LogConsumer interface {
 	// ConsumeLogs receives data.Logs for processing.
 	ConsumeLogs(ctx context.Context, ld data.Logs) error
 }
+
+// TestBedTraceConsumer is an interface that receives pdata.Traces,
+// and counts the number of traces received by the mock backend
+type TestBedTraceConsumer interface {
```

Comment:
Can this interface be in testbed?

Reply:
sorry I miss the comment I will check if we can add this interface in the testbed and will get back to you. thanks.